tests/single/gas/test_pre_verification_gas_calculation.py

```diff
@@ -107,4 +107,39 @@ def test_pre_verification_gas_calculation(
     expected_pre_vg = expected_min_pre_verification_gas[dynamic_length_field_name][
         field_length
     ]
+    # pylint: disable=fixme
+    # todo: tighten the 'approx' parameter
     assert min_pre_verification_gas == pytest.approx(expected_pre_vg, 0.1)
+
+
+# pylint: disable=fixme
+# todo: parametrize this test for different bundler 'expectedBundleSize' (requires new 'debug_' RPC API)
+def test_gas_cost_estimate_close_to_reality(wallet_contract, helper_contract):
```

Comment:
- prameterize on dynamic userop size (that is, validate the formula is valid for different sizes)
- need also setConfiguration ({ expectedBundleSize: xx})

Reply:
This method uses the above calculated table, but it is checked only against a single value.
slight modification to the userop size (e.g. sig size) and the test fails.

Reply:
Done.